controller/user-controller.js

```diff
@@ -0,0 +1,43 @@
+const userService = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname, skintype} =req.body;
+
+    
+
+    if (!email|| !password || !firstname || !lastname){
+      throw new Error("KEY Error!");
+    }
+
+    await userService.signUp(email, password, firstname, lastname, skintype);
+    return res.status(201).send({message: "SIGNUP_SUCCESS"});
+  } catch (err) {
+  res.status(err.statusCode || 400).json({message: err.message});
+  }
+};
+
+const signIn = async (req, res) => {
+ 
+  try{
+    const {email, password }= req.body;
+
+    const accessToken = await userService.signIn(email,password);
+   
+   
+    if( !email || !password ){
+    throw new Error("SIGN IN FAILED!");
+   }
```

Comment:
- 25번 라인 코드와 28~30번 라인의 코드의 순서가 바뀌어야 합니다.
이메일과 패스워드에 대한 데이터가 없을 때엔 `accessToken`을 발급하지 못하고 Key error를 반환해야 하기 때문입니다.

- 또한 불필요한 공백은 삭제해주세요. (26번 라인과 27번 라인 모두 공백일 필요는 없습니다. 코드를 구분하기 위해 공백을 두는 경우는 한줄만 띄워주세요.)

- 어제 구두로 설명드린 바와 같이 작성해주신 `if { ... }`  구문을 보면 같은 코드 블럭임에도 불구하고 들여쓰기가  맞지 않습니다. 해당 부분도 신경써서 수정해주세요.
```suggestion
   if( !email || !password ){
      throw new Error("SIGN IN FAILED!");
   }
   
    const accessToken = await userService.signIn(email,password);

```